frontend/src/assets/css/explorer/sidebar.css

```diff
@@ -5,7 +5,7 @@
     /* and footer also has 1 em padding */
     height: calc((100vh - 3.9em) - 3.1em);
     z-index: 1;
-    width: 30%;
+    width: 20%;
```

Comment:
https://gyazo.com/539311da67ee1dfab94ec34085f7dc69
Resolution kinda fucks up there.
also:
at this resolution you can't see west vlaanderen or luxemburg
https://gyazo.com/5ecfae3ee0b2029f08f939d7a05d0b2e

Reply:
We'll discuss responsiveness after we have basic filtering. You're right though